.travis.yml

```diff
@@ -24,14 +24,21 @@ stages:
 jobs:
   include:
   # patch formatting checks
-  - name: "Check patch formatting"
+  - name: "Python formatting"
     stage: check
     os: linux
     language: python
     python: 3.7
     before_install:
     install:
     script: ./scripts/ci/check_patch.sh
+  - name: "Script formatting"
+    os: linux
+    language: go
+    go: 1.13
+    before_install:
+    install:
+    script: ./scripts/ci/check_scripts.sh
```

Comment:
No need, just add that script to `scripts/ci/check_patch.sh`, where we check README and stuff like that anyways.

Reply:
Can't because it requires `go: 1.13` while the other scripts need a decent `python: 3.7`. Travis doesn't support having both in one job. I could get around this by manually imstalling both in one job (maybe) but not sure if it's worth the effort

Reply:
to be explicit, I already tried: https://github.com/iterative/dvc/compare/7a9512d02..0f13f4726 (that diff fixed travis)

Reply:
> nvm swapped to using `beautysh` which is python :tada: